### PR TITLE
FS-1231 RestX -> Connexion

### PR DESCRIPTION
**Pull request by SentientHousePlant**
✨  Removed RestX library and changed to Connexion, removed lots of classes and functions.

### CODE REVIEW

File: api/__init__.py [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
-    description="A simple Funding Service Design Application Store API",
-)
+# api = Api(
+#     title="Funding Service Design Application Store API",
+#     version="0.1.0",
+#     description="A simple Funding Service Design Application Store API",
+# )
 
-api.add_namespace(applications_ns)
+# api.add_namespace(applications_ns)
```
Comment (gidsg) [inline api/__init__.py:10]: Can we delete this commented out code?

File: app.py [hunk 2]
```diff
@@ -1,20 +1,69 @@
...
+#     health = Healthcheck(flask_app)
+#     health.add_check(FlaskRunningChecker())
+#     # TODO Add the following once PR 22 is merged (and we have a real DB)
+#     # health.add_check(DbChecker(db))
+#     return flask_app
+
+
+# app = create_app()
+
+
```
Comment (gidsg) [inline app.py:34]: Can we delete this commented out code?
```suggestion
```

File: api/routes/application/routes.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        try:
+            return_dict = get_application_with_forms(
+                uuid.UUID(application_id)
+            )
+            return return_dict, 200
+        except NoResultFound as e:
+            return {"code": 404, "message": str(e)}
+
+    def put(self):
+        # TODO : Whatever adams wants to to with this :shrug:
```
Comment (Adam-W1) [inline api/routes/application/routes.py:47]: Adam wants this to go :)

File: app.py [hunk 4]
```diff
@@ -1,20 +1,69 @@
...
+#     )
+
+#     health = Healthcheck(flask_app)
+#     health.add_check(FlaskRunningChecker())
+#     # TODO Add the following once PR 22 is merged (and we have a real DB)
+#     # health.add_check(DbChecker(db))
+#     return flask_app
+
+
+# app = create_app()
```
Comment (Adam-W1) [inline app.py]: ^ can go

File: tests/test_routes.py [hunk 5]
```diff
@@ -1,5 +1,6 @@
 import json
 from operator import itemgetter
+from pprint import pprint
```
Comment (gidsg) [inline tests/test_routes.py:3]: Looks line an unused dependency?

**Review by gidsg — APPROVED** [review]
LGTM 👍

**Review by Adam-W1 — APPROVED** [review]
Lovely, best thing Ive seen all year <3

**Pull request merged**